components/plugin_marketplace/marketplace_item/marketplace_item.js

```diff
@@ -119,9 +295,37 @@ export default class MarketplaceItem extends React.Component {
             pluginIcon = <PluginIcon className='icon__plugin icon__plugin--background'/>;
         }
 
+        let localTag;
+        if (!this.props.downloadUrl) {
+            const localTooltip = (
+                <Tooltip id='plugin-marketplace__local-tooltop'>
+                    <FormattedMessage
+                        id='marketplace_modal.list.local.tooltip'
+                        defaultMessage='This plugin is not listed in the marketplace but was installed manually.'
+                    />
+                </Tooltip>
+            );
+
+            localTag = (
+                <OverlayTrigger
+                    delayShow={Constants.OVERLAY_TIME_DELAY}
+                    placement='top'
+                    overlay={localTooltip}
+                >
+                    <span className='tag'>
+                        <FormattedMessage
+                            id='marketplace_modal.list.local'
+                            defaultMessage='Local'
```

Comment:
I'm wondering if `Local` is the right term here. 
1. To me it indicates that the plugin ins running locally, but this is misleading. There are also other plugins running locally that don't have this label.
2. It doesn't make clear why this plugin is listed there. It here because it was installed manually via a admin.

I'm wondering if it makes more sense to use `Manual` als an alternative. The help text could stay the same.


@asaadmahmood Let me know what you think about this

Reply:
> As discussed before, the way it was installed should have no relevance here but more the fact that it is only available on their server instance.

I don't think so. The feedback I have gotten was, that admins are confused by the plugins are listed there. That is the issue we want to address. And `local` does not give an answer to the question, why non marketplace plugins are listed there.

> Even though, `LOCAL` may be a bit unclear, I believe the popup message should clarify the situation and should be enough to clear any doubts on why is labeled that way.

Using this force, what separates `Local` from a more better name? If we rely on the popup message, we could use any arbitrary name. Have a self explaining name is important.

To apply our designe principles: `Local` is fast, but it is not obvious what it means.